docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -25,7 +25,7 @@ Parking is used widely throughout probing and tool changing to move the spindle
 G37
 ```
 
-When using multiple milling tools, we must compensate for length differences between the tools. G37 can be used to (re-)calculate the length of the current tool in relation to a reference surface or the previous tool.
+When using multiple milling tools, we must compensate for length differences between the tools. G37 can be used to (re-)calculate the length of the current tool about a reference surface or the previous tool.
```

Comment:
"In relation to" seems clearer here

Reply:
Agreed, about is fine when talking about rotation but in this case it is a relative length, calculated with the difference in height between the reference surface and toolsetter activation point.